app/models/validations/sales/sale_information_validations.rb

```diff
@@ -23,4 +23,13 @@ def validate_previous_property_unit_type(record)
       record.errors.add :fromprop, I18n.t("validations.sale_information.previous_property_type.property_type_bedsit")
     end
   end
+
+  def validate_basic_monthly_rent(record)
+    return unless record.mrent && record.ownershipsch && record.type
+
+    if record.shared_owhership_scheme? && !record.old_persons_shared_ownership? && record.mrent > 9999
```

Comment:
I would expect rubocop would complain about 9999 and say it needs to be 9_999 
does it only start complaining after 5 digits or more?

Reply:
I think 4 9s is reasonable to read